content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "EC3 applications and tools"
+type: docs
+weight: 30
+description: >
+  How to run scientific applications and tools with EC3
+---
+
+This section documents how to run some applications and use the existing tools
+with EC3.
+
+## How to run scientific applications in EC3
+
+### NAMD cluster
+
+To deploy [NAMD](https://www.ks.uiuc.edu/Research/namd/) clusters, please select one of the available LRMS (Local Resource
+Management System) and choose NAMD from the list of applications.
+
+## How to use generic tools/practices in EC3
+
+### Apache Mesos
```

Comment:
@glarocca and should this one be deleted to?

Reply:
To have only the entry with Mesos + Chronos + Marathon?

Reply:
yes, I removed in one of my last commit.

Reply:
I think you renamed the Marathon one in https://github.com/EGI-Foundation/documentation/pull/151/commits/1f74e4709cf2de679e6f6fde9b1ffeb7ba4f9acd and removed the Chronos one in https://github.com/EGI-Foundation/documentation/pull/151/commits/87056cca2c48b3068067122364a96d738e152df1, but the Mesos one seems still present.